docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -1,70 +1,66 @@
 
 ~~~~~~~~~~~~~~{.cpp}
 #include <dpp/dpp.h>
-#include <fmt/format.h> // used to format text, install and define FMT_HEADER_ONLY to use
 
-struct GetMessage {
-    dpp::snowflake message_id = 1146725845457719367ULL;
-    dpp::snowflake channel_id = 1112789762521182211ULL;
-};
+const std::string BOT_TOKEN = "Token Was Here";
 
 int main() {
-    GetMessage gm;
-
-    std::string BOT_TOKEN = "Token Was here";
-
-    /* the second argument is a bitmask of intents - i_message_content is needed to get messages */
-    dpp::cluster bot(BOT_TOKEN, dpp::i_message_content | dpp::i_default_intents);
+    dpp::cluster bot(BOT_TOKEN, dpp::i_default_intents | dpp::i_message_content);
+    // the second argument is a bitmask of intents - i_message_content is needed to get messages
 
     bot.on_log(dpp::utility::cout_logger());
 
-    bot.on_slashcommand([&bot, gm](const dpp::slashcommand_t& event) -> void {
-        if (event.command.get_command_name() == "msg-get") {
+    // The event is fired when someone issues your commands
+    bot.on_slashcommand([&bot](const dpp::slashcommand_t& event) -> void {
+        if (event.command.get_command_name() == "msgs-get") {
+            int64_t limit = std::get<int64_t>(event.get_parameter("quantity"));
 
-            // get a message using its own and channel's ID
-            bot.message_get(gm.message_id, gm.channel_id, [&bot, event](const dpp::confirmation_callback_t& callback) -> void {
+            // get messages using ID of the channel the command was issued in
+            bot.messages_get(event.command.channel_id, 0, 0, 0, limit, [event](const dpp::confirmation_callback_t& callback) -> void {
                 if (callback.is_error()) { // catching an error to log it
-                    bot.log(dpp::loglevel::ll_error, callback.get_error().message);
+                    std::cout << callback.get_error().message << std::endl;
                     return;
                 }
 
-                // callback.value will contain the message we are getting but the type will be std::variant, so we will have to get the message itself.
-                auto message = callback.get <dpp::message>();
-                //std::get <dpp::message>(callback.value) would give the same result
+                auto messages = callback.get<dpp::message_map>();
+                // std::get <dpp::message_map>(callback.value) would give the same result
```

Comment:
`// std::get<dpp::message_map>`! Need to remove that space between `get` and `dpp::message_map`

Reply:
i forgor

Reply:
don